include/aunit/reporters/aunit-reporter-combine.ads

```diff
@@ -0,0 +1,26 @@
+with Ada.Containers.Vectors;
+
+package AUnit.Reporter.Combine is
+
+   type Combined_Reporter is new Reporter with private;
+
+   overriding
+   procedure Report
+     (Engine  : Combined_Reporter;
+      R       : in out Result'Class;
+      Options : AUnit_Options := Default_Options);
+
+   procedure Add_Reporter (C : in out Combined_Reporter; R : access constant Reporter'Class);
```

Comment:
R should be of type Reporter_Access, otherwise there would be a compilation error:

aunit-reporter-combine.adb:20:26: implicit conversion of anonymous access parameter not allowed